README.md

```diff
@@ -3,3 +3,5 @@
 This project demonstrates the recommended integration approach for Bluedot Point SDK.  To use this example project, you first need to have active Bluedot account and use the credentials. To request for a demo please visit [Request Demo] https://bluedotinnovation.com/demo.
 
 For more developer documentation please visit the [Bluedot Developer Documentation](http://docs.bluedotinnovation.com/display/DEVDOC10/Android+SDK).
```

Comment:
These urls are very old, could you update to `https://bluedot.io/demo/` for the demo link and `https://docs.bluedot.io/android-sdk` for dev docs?